sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -172,19 +183,24 @@ def begin_recognize_content_from_url(self, url, **kwargs):
 
     @distributed_trace
     def begin_recognize_custom_forms(self, model_id, stream, **kwargs):
-        # type: (str, IO[bytes], Any) -> LROPoller
-        """Analyze Form.
+        # type: (str, Union[bytes, IO[bytes]], Any) -> LROPoller
+        """Analyze a custom form with a model trained with or without labels.
+        The input document must be of one of the supported content types - 'application/pdf',
+        'image/jpeg', 'image/png' or 'image/tiff'.
 
         :param str model_id: Model identifier.
         :param stream: .pdf, .jpg, .png or .tiff type file stream.
         :type stream: stream
         :keyword bool include_text_content: Include text lines and element references in the result.
-        :keyword str content_type: Media type of the body sent to the API. For options,
+        :keyword str content_type: Media type of the body sent to the API. Content-type is
+            auto-detected, but can be overridden by passing this keyword argument. For options,
             see :class:`~azure.ai.formrecognizer.FormContentType`.
-        :return: LROPoller
+        :return: An instance of an LROPoller. Call `result()` on the poller
```

Comment:
I'd make this more specific to "Call `result()` ... to return the list of `RecognizedForm`s